Implement real fix for C5271 in IJW test build

Recent VS upgrade introduces C5271, and causes a build break like this:

```
src\native\corehost\test\ijw\ijw.cpp(6): warning C5271: consider replacing #using <System.Console.dll>  with command line argument /FU "F:\dotnet\runtime2\.dotnet\packs\Microsoft.NETCore.App.Ref\8.0.0-rc.1.23414.4\ref\net8.0\System.Console.dll"
src\native\corehost\test\ijw\ijw.cpp(7): warning C5271: consider replacing #using <System.Runtime.Loader.dll>  with command line argument /FU "F:\dotnet\runtime2\.dotnet\packs\Microsoft.NETCore.App.Ref\8.0.0-rc.1.23414.4\ref\net8.0\System.Runtime.Loader.dll"
```

https://github.com/dotnet/runtime/pull/92878 disables the warning. This issue is to track implementing a real fix for this and removing the warning disable.

--- a/eng/native/ijw/IJW.cmake
+++ b/eng/native/ijw/IJW.cmake
@@ -45,7 +45,7 @@ if (CLR_CMAKE_HOST_WIN32)
 
   # 4365 - signed/unsigned mismatch
   # 4679 - Could not import member. This is an issue with IJW and static abstract methods in interfaces.
-  add_compile_options(/wd4365 /wd4679 /wd5271)
+  add_compile_options(/wd4365 /wd4679)
 
   # IJW
   add_compile_options(/clr:netcore)
@@ -92,6 +92,12 @@ if (CLR_CMAKE_HOST_WIN32)
   string(REGEX REPLACE ".*refPackPath=(.*)" "\\1" CLR_SDK_REF_PACK ${CLR_SDK_REF_PACK_OUTPUT})
 
   add_compile_options(/AI${CLR_SDK_REF_PACK})
+
+  file(GLOB CLR_SDK_REF_PACK_LIBS "${CLR_SDK_REF_PACK}/*.dll")
+
+  foreach(lib ${CLR_SDK_REF_PACK_LIBS})
+    add_compile_options(/FU${lib})
+  endforeach()
 
   list(APPEND LINK_LIBRARIES_ADDITIONAL ijwhost)
 

--- a/src/native/corehost/test/ijw/ijw.cpp
+++ b/src/native/corehost/test/ijw/ijw.cpp
@@ -3,9 +3,6 @@
 
 #include <iostream>
 
-#using <System.Console.dll>
-#using <System.Runtime.Loader.dll>
-
 using namespace System;
 using namespace System::Reflection;
 using namespace System::Runtime::Loader;

--- a/src/tests/Interop/IJW/NativeVarargs/IjwNativeVarargs.cpp
+++ b/src/tests/Interop/IJW/NativeVarargs/IjwNativeVarargs.cpp
@@ -8,8 +8,6 @@
 #include <array>
 #include <functional>
 #include <iostream>
-#using <System.Runtime.dll>
-#using <System.Collections.dll>
 using namespace System::Collections::Generic;
 
 public enum class TestCases

